stars/default/name.fab

```diff
@@ -1309,6 +1448,7 @@
  27628|β_Col
  28199|γ_Col
  30277|δ_Col
+ 30277|3_CMa*
```

Comment:
Why asterisk added here?

Reply:
according to [δ Col](https://en.wikipedia.org/wiki/Delta_Columbae) and [Table of stars with Flamsteed designations](https://en.wikipedia.org/wiki/Table_of_stars_with_Flamsteed_designations)

Reply:
This star hasn’t asterisk in designation!